src/qt/guiutil.h

```diff
@@ -233,19 +230,7 @@ namespace GUIUtil
         void mouseReleaseEvent(QMouseEvent *event);
     };
 
-#if defined(Q_OS_MAC)
```

Comment:
Has someone verified that this BUG no longer appears on OSX 10.10+?

Reply:
@jonasschnelli I haven't been able to recreate it yet, will test further though.

Reply:
The new URL for the bug appears to be https://bugreports.qt.io/browse/QTBUG-15631 but that bug seems unrelated to the comment.

Reply:
@kallewoof I mentioned in the PR description, from what I can tell the correct link is https://bugreports.qt.io/browse/QTBUG-20880